Gemfile.lock

```diff
@@ -104,55 +104,55 @@ GEM
       mini_mime (>= 0.1.1)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
+    method_source (1.0.0)
```

Comment:
### [method_source](http://banisterfiend.wordpress.com)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.9.2 → 1.0.0

[_~~change-log~~, [source-code](https://github.com/banister/method_source)_]





<details>
<summary>Commits</summary>

A change of **17** commits. See the full changes on [the compare page](https://github.com/banister/method_source/compare/v0.9.2...v1.0.0).

These are the first 10 commits:
- (811ab69) [Fix `documentation` link](https://github.com/banister/method_source/commit/811ab6914558e18f566da801ebf1810777d3dfbb)
- (7079cd7) [Merge pull request #56 from nisusam/fix_documentation_link](https://github.com/banister/method_source/commit/7079cd72e320e689abfe9d60b164553b98b5eeaf)
- (de0d668) [Fix CI build](https://github.com/banister/method_source/commit/de0d668039cc4df277eb0191f464332e4fa25a0a)
- (f6e4488) [Add MRI 2.5 and 2.6](https://github.com/banister/method_source/commit/f6e4488b8fcab1a5648de145f89669ad27f9619a)
- (6487a1e) [Fix ruby warning in spec_helper](https://github.com/banister/method_source/commit/6487a1e425997f674c6a6bb8c9e5e83c425ba271)
- (f7cdccb) [Merge pull request #60 from casperisfine/fix-ci](https://github.com/banister/method_source/commit/f7cdccb814f9dcc72a7075a4d548b4568c90285b)
- (5461b17) [Handle new message for unterminated lists on MRI 2.7](https://github.com/banister/method_source/commit/5461b17602b465f8d259e25f0e749c4dff8b6208)
- (a7334bb) [Test against Ruby 2.7 on CI](https://github.com/banister/method_source/commit/a7334bbcc36bcb60b3bb42f796bd72af7828bb82)
- (24ccd7f) [Merge pull request #59 from casperisfine/fix-ruby-2.7](https://github.com/banister/method_source/commit/24ccd7fa619245abddeb03fcf40c8cb452dc5666)
- (b983b28) [More closely match MIT License text verbatim](https://github.com/banister/method_source/commit/b983b2805279ffa706db90998520276888634c2a)


</details>